.astyleignore

```diff
@@ -1,9 +1,9 @@
 ^BUILD
 ^cmsis
+^connectivity/FEATURE_BLE
```

Comment:
the old check was for features/FEATURE_BLE so I figured I'd better move it as is since I don't want to get bogged down with astyle right now, we can fix this in a later PR